### PR TITLE
Remember email in sessionStorage

### PR DESCRIPTION
**Pull request by jamespamplin**
Remembers the user's email address in browser `sessionStorage` instead of using a URL parameter.

### CODE REVIEW

**Comment by jamespamplin** [comment]
Fixed merge conflicts due to  #44

File: public/components/browser/storage.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * Basic abstraction around browser key/value storage, eg: sessionStorage and localStorage.
+ *
+ * @param {string} type
+ */
+function storage( type ) {
+  const storageActual = window[ type ];
+
+  function checkSupported() {
```
Comment (markjamesbutler) [inline public/components/browser/storage.js]: isSupported would read a little better.

**Comment by markjamesbutler** [comment]
LGTM :+1:

File: public/components/browser/storage.js (hunk 1 above)
Comment (jamespamplin) [inline public/components/browser/storage.js]: I've called this `checkSupported()` as it throws an error if it fails. `isSupported` would be ok if its just a boolean check.

Its unlikely for this code to throw as I have a high-level check for `sessionStorage` in `public/components/browser/browser.js`.

For now I think this is ok.

**Pull request merged**